transcriptic/commands.py

```diff
@@ -913,7 +924,8 @@ def login(api, config, api_root=None, analytics=True):
                 '200': lambda resp: resp,
                 '401': lambda resp: resp,
                 'default': lambda resp: resp
-            }
+            },
+            auth=StrateosSign(email, rsa_secret)
```

Comment:
do we need to add this line to the organization get call below as well?

Reply:
yes, done

Reply:
I wasn't going thought the org select flow because I'm only a collab on one org, so that appears to be the reason it was working for me but not you.